### DISCUSSION TITLE
About vulnerabilities json-smart@1.3.1 on image docker

### DISCUSSION THREAD

**Discussion by tsukiazuma** [aquasecurity/trivy#4875] [Q&A]
### Question

Hi,

I have problem about vulnerabilities json-smart@1.3.1 on image docker:
- Source code not found where json-smart@1.3.1 used
- Image not have vulnerabilities before build with code
But deploy code into image and scan with Trivy, it warning about json-smart@1.3.1

Is it false positive? If not, how can I do to fix it. Please

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

### Operating System

Linux

### Version

```bash
v0.42.0
```

**Comment by knqyf263** [comment]
Please try `-f json` and see the detail

**Reply by tsukiazuma** [reply]
Oh sorry, I scan with default trivy on Harbor, so can't use `-f json`

**Reply by DmitriyLewen** [reply]
Maybe you can install Trivy on your local PC and scan your image in `json` format?
